geo/src/algorithm/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,36 @@
+# The Rust-Geo Code of Conduct
```

Comment:
Is our CoC going to cover all GeoRust projects? If so, should this be the GeoRust CoC?

Reply:
I think that's a good idea – should we ask individual project owners?